templates/buy-me-a-coffee-app/src/pages/_app.tsx

```diff
@@ -1,14 +1,22 @@
-import '@/styles/globals.css';
+import '@radix-ui/themes/styles.css';
+import '../styles/globals.css';
 import '@rainbow-me/rainbowkit/styles.css';
 
 import type { AppProps } from 'next/app';
-
+import { ThemeProvider } from 'next-themes';
+import { Theme } from '@radix-ui/themes';
 import Web3Providers from './Web3Providers';
 
+const themeValues = { light: 'light-theme', dark: 'dark-theme' };
+
 export default function App({ Component, pageProps }: AppProps) {
   return (
-    <Web3Providers>
-      <Component {...pageProps} />
-    </Web3Providers>
+    <ThemeProvider disableTransitionOnChange attribute="class" value={themeValues}>
+      <Theme accentColor="orange">
+        <Web3Providers>
```

Comment:
Just a thought, should this be renamed.... BuildOnchainProvider, and, and, this provider is one of the first Kit?

After all it feels one of the main recipe parts.